### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
in this project i added a contact form using input element  the following elements were used to build this contact form
-form elements 
-label elements
-input elements
-textarea and 
-button elements 
Languages used to realized this were 
-HTML  and 
-CSS  to style the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,215 +4,230 @@
...
-        </section> 
-     </main>
-    </body>
-     </html>
-     
\ No newline at end of file
+                </section>
+                                <div class="flex-container"> 
+                <section class="contactform1">
+                    <h1>Contact me</h1>
```
Comment (OmarRamoun) [inline index.html:219]: - [ ] [REQUIRED] Please, make sure to always use only one `h1` element in the whole document for better SEO Ranking. You can check for "semantic h1 uses`

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @Ngufor-emmanuella 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,215 +4,230 @@
...
-                <p><a href="#">My Logo</a></p>
+    <header class="header"> 
+        <a class="logo"href="#">My Logo</a>
+        <nav class="nav">
+            <img class="ham" src="images/menu-icon.svg" alt="menu">
+            <!--<ul class="navbar">
+                <li><a class="navlinks" href="#">portfolio</a></li>
+                <li><a class="navlinks" href="#">about</a></li>
+                <li><a class="navlinks"href="#">contact</a></li> 
+            </ul>-->
```
[the remark this answers is not in the document]
Comment (Ngufor-emmanuella) [reply index.html]: thanks for the corrections

File: index.html (hunk 1 above)
Comment (Ngufor-emmanuella) [reply index.html]: corrected it thanks

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Ngufor-emmanuella 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**